rumbler/rumbler-portfolio
# fix: use --follow-tags when pushing version tags

## Description
Modifies the version bump script to use --follow-tags when pushing tags to the remote repository.

### Problem
The CD workflow (production.yml) wasn't being triggered when a tag was created via script.

### Solution
- Changed the push command to use `--follow-tags`
- Now we push the tag along with the commit using `git push --follow-tags origin main`

### Impact
- GitHub Actions will now properly recognize the tag event
- Production workflow will be triggered as expected

### Testing
1. Merge this PR
2. Checkout to main
3. Run `pnpm version:tag`
4. Verify that the production workflow is triggered
